hover_height: 2.0
recording_data: False
predefined_goal: [[13, 6], [13, -6], [0, 0]] # this is only used when mission_mode is 'predefined'
